nbconvert/exporters/html.py

```diff
@@ -27,6 +29,58 @@
 from .templateexporter import TemplateExporter
 
 
+def find_lab_theme(theme_name):
+    """
+    Find a JupyterLab theme location by name.
+
+    Parameters
+    ----------
+    theme_name : str
+        The name of the labextension theme you want to find.
+
+    Raises
+    ------
+    ValueError
+        If the theme was not found, or if it was not specific enough.
+
+    Returns
+    -------
+    theme_name: str
+        Full theme name (with organization any, if any)
+    labextension_path : Path
+        The path to the found labextension on the system.
+    """
+    paths = jupyter_path('labextensions')
+
+    matching_themes = []
+    theme_path = None
+    for path in paths:
+        for (dirpath, dirnames, filenames) in os.walk(path):
+            # If it's a federated labextension that contains themes
+            if 'package.json' in filenames and 'themes' in dirnames:
+                # TODO Find the theme name in the JS code instead?
+                # TODO Find if it's a light or dark theme?
```

Comment:
I think it would be fine to leave these TODOs here for now, so this PR can be merged and iterated on later :+1:

Reply:
Yes I guess that's the best we can do for now :)